src/public/images/transitions.svg

```diff
@@ -21,6 +21,16 @@
         fill: #231f20;
       }
 
+      @media (prefers-color-scheme: dark) {
```

Comment:
May I have more information and background about this change?

Reply:
Oh, yes, I tried to do some kind of enhancement to the dark mode presentation of this SVG file because the current one is a little strange. This is a colour modification of the SVG file based on the user's colour scheme preferences.

Below, you can see what I was trying to do, @Jinjiang

| current dark mode | new dark mode in the pull request |
| --------- | ------- |
| ![vue3-migration-guide-old-dark-theme](https://github.com/vuejs/v3-migration-guide/assets/31008635/3b145e81-754e-4020-a448-5ee7ea684d09) | ![vue3-migration-guide-new-dark-theme](https://github.com/vuejs/v3-migration-guide/assets/31008635/1178ac84-5fd0-4841-a41e-4bc052a3bb7d) |

Reply:
Gotcha. Great job. Just would be better if it could be a separated PR next time. :-)
I will approve and merge it shortly. Thanks.